chain/wallet/wallet.go

```diff
@@ -160,6 +161,24 @@ func (w *LocalWallet) WalletImport(ctx context.Context, ki *types.KeyInfo) (addr
 	return k.Address, nil
 }
 
+func (w *LocalWallet) WalletMsigImport(ctx context.Context, idAddress address.Address, robustAddress address.Address) error {
+	log.Infof("WalletMsigImport idAddress:%s, robustAddress:%s", idAddress.String(), robustAddress.String())
+
+	w.lk.Lock()
+	defer w.lk.Unlock()
+
```

Comment:
>there should be a check to make sure that whoever is importing the msig has the key to one of the msig signers, because later we also have to pick which of the signer addresses to use for msig operations

same here